### DISCUSSION TITLE
【路由前缀】NEXT_PUBLIC_POST_URL_PREFIX修改为空值，URL错误的问题

### DISCUSSION THREAD

**Discussion by maokongrun** [tangly1024/NotionNext#1987] [Q&A]
**描述bug**
简单说明bug的现象、相关的错误提示、日志等
在Vercel的环境变量中设置：NEXT_PUBLIC_POST_URL_PREFIX为''，但是URL变成是https://xxx.vercel.app/''/slug，而不是https://xxx.vercel.app/slug

**截图**
相关的页面，应该的结果

<img width="1055" alt="image" src="https://github.com/tangly1024/NotionNext/assets/26140353/db25d6cb-0ef6-4508-9850-2af984620356">
<img width="327" alt="image" src="https://github.com/tangly1024/NotionNext/assets/26140353/d8af3f12-5272-4a74-b85d-82cfe12d6746">

**环境**

- 操作系统: macOS
- 浏览器 Chrome
- NotionNext版本：4.0.15
- 主题 [例如. hexo]

**Comment by tangly1024** [comment]
留`空白`试一下，不要填写 `''`这个符号
